cypress/features/regression/sidebar.feature

```diff
@@ -7,7 +7,6 @@ Feature: Sidebar component
 
   @positive
   Scenario: CloseIcon has the border outline
```

Comment:
Should be always `Given` - `When` - `Then` of `When` - `Then`
``` suggestion
Scenario: CloseIcon has the border outline
# commented because of BDD default scenario Given - When - Then
# When I check open checkbox
```
We need to comment `When` here - because the step was actually performed in `Background` and add a comment why it was commented out